tests/urlpatterns_reverse/tests.py

```diff
@@ -361,6 +361,30 @@ def test_resolver_reverse(self):
         self.assertEqual(resolver.reverse('named-url2', 'arg'), 'extra/arg/')
         self.assertEqual(resolver.reverse('named-url2', extra='arg'), 'extra/arg/')
 
+    def test_resolver_reverse_conflict(self):
+        """
+        Verifies that url names can indeed conflict: they need not be unique.
+
+        Behavior is that the last registered pattern will take precedence
+        for direct conflict of patterns.
+
+        Clarification tests for #27367
+        """
+        resolver = get_resolver('urlpatterns_reverse.named_urls_conflict')
+
+        # no argument test
```

Comment:
The three comments can be removed. Leaving a blank line between the "test sets" is OK.